src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -366,11 +380,13 @@ private async ValueTask<ProtocolToken> ReceiveBlobAsync<TIOAdapter>(TIOAdapter a
 
             if (_lastFrame.Header.Length < 0)
             {
+                NetEventSource.Error(this, "invalid TLS frame size");
```

Comment:
Ditto

Reply:
added for consistently. 

https://github.com/dotnet/runtime/blob/e2ed3631b9a2a97de10991e4b548f63086cdc67a/src/libraries/Common/src/System/Net/Logging/NetEventSource.Common.cs#L31-L38

and

https://github.com/dotnet/runtime/blob/e2ed3631b9a2a97de10991e4b548f63086cdc67a/src/libraries/Common/src/System/Net/Logging/NetEventSource.Common.cs#L238-L242

has duplicate check.